search-app/src/cmr/search/services/acls/collections_cache.clj

```diff
@@ -1,151 +1,19 @@
-   data))
-
-;; added to common lib will remove in future
-(defn- fetch-collections
-  "Executes a query that will fetch all of the collection information needed for caching."
-  [context]
-  ;; when creating a result processor, realize all the lazy (delay) values to
-  ;; actual values so that the resulting object can be cached in redis or some
-  ;; other text based caching system and not clojure memory
-  (let [result-processor (fn [_ _ elastic-item]
-                           (assoc (util/delazy-all (acl-rhh/parse-elastic-item
-                                                    :collection
-                                                    elastic-item))
-                                  :concept-id (:_id elastic-item)))
-        query (q-mod/query {:concept-type :collection
-                            :condition q-mod/match-all
-                            :skip-acls? true
-                            :page-size :unlimited
-                            :result-format :query-specified
-                            :result-fields (cons :concept-id acl-rhh/collection-elastic-fields)
-                            :result-features {:query-specified {:result-processor result-processor}}})]
-    (:items (qe/execute-query context query))))
-
-;; added to common lib will remove in future
-(defn- fetch-collections-map
-  "Retrieve collections from search and return a map by concept-id and provider-id"
-  [context]
-  (let [collections (fetch-collections context)
-        by-concept-id (into {} (for [{:keys [concept-id] :as coll} collections]
-                                 [concept-id coll]))
-        by-provider-id-entry-title (into {}
-                                         (for [{:keys [provider-id EntryTitle] :as coll}
-                                               collections]
-                                           [[provider-id EntryTitle] coll]))]
-    ;; We could reduce the amount of memory here if needed by only fetching the collections that
-    ;; have granules.
-    {:by-concept-id by-concept-id
-     :by-provider-id-entry-title by-provider-id-entry-title}))
-
-;; added to common lib will remove in future
-(defn refresh-cache
-  "Refreshes the collections stored in the cache. This should be called from a background job on a timer
-  to keep the cache fresh. This will throw an exception if there is a problem fetching collections. The
-  caller is responsible for catching and logging the exception."
-  [context]
-  (let [cache (hash-cache/context->cache context cache-key)
-        collections-map (fetch-collections-map context)]
-    (doseq [[coll-key coll-value] collections-map]
-      (hash-cache/set-value cache cache-key coll-key (clj-times->time-strs coll-value)))))
+    [cmr.common.hash-cache :as hash-cache]
+    [cmr.common.log :as log :refer (debug info warn error)]
+    [cmr.common-app.data.search.collection-for-gran-acls-caches :as coll-caches]))
 
 (defn get-collection-for-gran-acls
   ([context coll-concept-id]
```

Comment:
It looks like the get-collection-for-gran-acls context provider-d entry-title is not being used.

In that case on new line 9 remove the extra ( and one at the end of this function as you don't need them anymore.

Reply:
will do